docs/en/index.md

```diff
@@ -6,109 +6,109 @@
-our [R-Dashboards GitHub repository](https://github.com/StatCan/R-dashboards).
-![R Shiny Server](images/readme/shiny_ui.png)
+!!! note ""
+    <center>
+    [![Kubeflow is the Heart of the AAW!](./images/Kubeflow.PNG)](https://kubeflow.aaw.cloud.statcan.ca/)
+    <h3>**[👉 Click here to setup your Kubeflow account! 👈](https://kubeflow.aaw.cloud.statcan.ca/)**</h3>
+    </center>
 
-**[Dash](/2-Publishing/Dash/)** is a data visualization tool that lets you build
-an interactive GUI around your data analysis code.
+**[Kubeflow](1-Experiments/Kubeflow/)** is a powerful and flexible open source platform that has revolutionized the way we build, deploy, and manage machine learning workflows. With its robust set of tools and frameworks, Kubeflow is the perfect starting point for data scientists and developers who want to accelerate their projects and bring them to the next level. By using up Kubeflow, you can harness the full potential of cloud-native machine learning and unlock new opportunities for innovation and growth.
 
-## Explore your data
+!!! important "Attention Statistics Canada Employees!"
+    Users internal to Statistics Canada can get started right away without any additional sign up procedures, just head to  [https://kubeflow.aaw.cloud.statcan.ca/](https://kubeflow.aaw.cloud.statcan.ca/).
 
-[![Explore your data](images/ExploreData.PNG)](/2-Publishing/Datasette/) Use
-**[Datasette](/2-Publishing/Datasette/)** , an instant JSON API for your SQLite
-databases. Run SQL queries in a more interactive way!
+Kubeflow offers an unparalleled level of flexibility and customization, empowering you to create and deploy machine learning workflows that meet your specific needs and requirements. By embracing Kubeflow, you'll be at the forefront of the machine learning revolution, and who knows what incredible insights and breakthroughs you'll discover along the way.
 
-# Pipelines
+### Slack
+
+<center>
+[![Ask questions on Slack](images/SlackAAW.PNG)](https://statcan-aaw.slack.com/)
+</center>
 
-## Build and schedule data/analysis pipelines
+- **[Click here sign in to our Slack Support Workspace](https://statcan-aaw.slack.com/)**
 
-[![Build Piplines](images/BuildPipelines.PNG)](/3-Pipelines/Kubeflow-Pipelines/)
-**[Kubeflow Pipelines](/3-Pipelines/Kubeflow-Pipelines/)** allows you to set up
-pipelines. Each pipeline encapsulates analytical workflows, and can be shared,
-reused, and scheduled.
-![Kubeflow Pipelines](images/readme/kubeflow_pipeline.png)
+- **Use the _General_ Channel!**
 
-[![Integrate with PaaS](images/IntegratePaaS.PNG)]()
+At StatCan, we understand that embarking on a new project can be overwhelming, and you're likely to have many questions along the way. That's why we've created a dedicated  **[Slack channel](https://statcan-aaw.slack.com/)** to provide you with the support you need. Our team of experts is standing by to answer your questions, address any concerns, and guide you through every step of the process.
```

Comment:
Platform related questions. Just limiting the scope because people sometimes ask dev questions.